packages/core/src/analytics.ts

```diff
@@ -272,6 +273,8 @@ export class SegmentClient {
 
       // flush any stored events
       this.flushPolicyExecuter.manualFlush();
+
+      await Inspector.init();
```

Comment:
Can we make this enable only when running while on debug? You can use `this.config.debug` to check